core/templates/dev/head/domain/feedback_message/FeedbackMessageSummaryObjectFactory.js

```diff
@@ -30,24 +30,24 @@ oppia.factory('FeedbackMessageSummaryObjectFactory', [function() {
     this.createdOn = createdOn;
   };
 
-  FeedbackMessage.createNewMessage = function(newMessage, authorUsername,
+  FeedbackMessageSummary.createNewMessage = function(newMessage, authorUsername,
     authorPictureDataUrl) {
-    return new FeedbackMessage(
+    return new FeedbackMessageSummary(
       newMessage, null, null, null, authorUsername, authorPictureDataUrl,
       new Date());
   };
 
-  FeedbackMessage.createFromBackendDict = function(
-    feedbackMessageBackendDict) {
-    return new FeedbackMessage(
-      feedbackMessageBackendDict.text,
-      feedbackMessageBackendDict.suggestion_html,
-      feedbackMessageBackendDict.current_content_html,
-      feedbackMessageBackendDict.description,
-      feedbackMessageBackendDict.author_username,
-      feedbackMessageBackendDict.author_picture_data_url,
-      feedbackMessageBackendDict.created_on);
+  FeedbackMessageSummary.createFromBackendDict = function(
+    FeedbackMessageSummaryBackendDict) {
```

Comment:
The arg should start with a lowercase letter: feedbackMessageSummaryBackendDict.